build.sh

```diff
@@ -4,8 +4,10 @@ basedir=$(cd $(dirname "$0"); pwd)
 
 source $basedir/setup.sh
 
+platforms="linux,windows,web,android,osx,ios,msvc"
```

Comment:
The `msvc` platform doesn't work currently, so it shouldn't be built by default. That's why it's disabled in `main`.